tests/testthat/test_features.R

```diff
@@ -359,5 +369,14 @@ test_that("allFeatures works as intended", {
   expect_equal(ncol(dt), 15)
   expect_equal(nrow(dt), 4)
 
+  # only nonvarying by Ribo-seq
+  dt <- ORFik:::allFeatures(grl = grl,orfFeatures = T, RFP = RFP5GAlign, RNA = RNAGAlign,
+                            tx = tx, fiveUTRs = fiveUTRs, cds = cds,
+                            threeUTRs = threeUTRs, riboStart = 26, riboStop = 34,
+                            extension = 5, cageFiveUTRs = fiveUTRs, includeNonVarying = F)
+  expect_is(dt, "data.table")
+  expect_equal(ncol(dt), 10)
```

Comment:
this test is a start, check maybe also whether all features are equal to the saved values.